### PR TITLE
feat: add support for sway inventory

### PR DESCRIPTION
**Pull request by Lazerbeak12345**
adds this gui as an inventory tab on sway

### CODE REVIEW

**Comment by Lazerbeak12345** [comment]
I'm noticing that the verbana gui tabs are _very_ slow to respond.

I've found that this is only a problem when prefixed. It happens both inside sway, and with show-ed (but with a prefix) like so:

```lua
        flow.make_gui(function ()
            return verbana_gui_form:embed {
                player = player,
                name = "verbana",
            }
        end):show(player)
```

This is most likely a bug with flow - though an easy workaround is to return true from the `on_event` handler for the `gui.Tabheader`

I'll mark this a draft till I report that bug to flow, and it is resolved.

**Comment by luk3yx** [comment]
I think I know what has happened. https://github.com/luk3yx/minetest-flow/blob/main/embed.lua#L25 stores a reference to `ctx.form`, but this reference won't get updated when flow wraps the form object to detect indexing (so the form won't get updated when you switch tabs). Perhaps it'd be better to store ctx instead?

**Comment by luk3yx** [comment]
I've updated flow and I think it works properly now, but I haven't tested this PR with it (I don't have verbana set up, I've actually only tested this mod on Blocky Survival and have never set it up locally).

File: init.lua [hunk 1]
```diff
@@ -599,3 +599,43 @@ minetest.register_chatcommand("verbana_gui", {
...
+        if priv ~= verbana.privs.moderator then
+            return
+        end
+
+        local player = minetest.get_player_by_name(player_name)
+        if not player then
+            return
+        end
+
+        if sway.get_page(player) ~= pagename then
```
Comment (Lazerbeak12345) [inline init.lua:633]: Just re-read this. This needs to be `pagename..":gui"` to match how it's registered in sway.

**Pull request merged**